src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -1010,6 +1012,348 @@ static bool processAreaScope(Scope* scoP, BSONObj &areaQuery)
   return true;
 }
 
+/* ****************************************************************************
+*
+* qStringFilters -
+*
+* FIXME P9: this function currently abuses of the std::string() wrapping for char* in order to
+* make comparisons work. An smarter solution could be developed.
+*
+*/
+static void qStringFilters(std::string& in, std::vector<BSONObj> &filters)
```

Comment:
It is too "tied" with the query functionality to live in a file of it own. What probably is a better idea is to package all the functions related with query (basically, the functions that entityQuery() calls in a separate file). In general, MongoGlobal is too large and there are functions groups that could be moved outside.

Do you think is a good moment to do it know? Any potential conflict in another queued PR?